code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -710,6 +710,11 @@
 	..()
 
 /datum/reagent/medicine/epinephrine/on_mob_life(mob/living/carbon/M)
+	if(holder.has_reagent(/datum/reagent/toxin/lexorin))
+		holder.remove_reagent(/datum/reagent/toxin/lexorin, 3)
+		holder.remove_reagent(/datum/reagent/medicine/epinephrine, 2)
+		if(prob(20))
+			holder.add_reagent(/datum/reagent/toxin/histamine, 4)
```

Comment:
bring 728 up and early return if you have lexorin in your system so if you counter lexorin this way it opens you up to other issues on top of the histamine. Also make it TRUE, it's a boo.

I'd like for it to be less steep in purging both, 3u on iteration is 20% of a syringe gun. Just making it 1 (and not even purging epi) would still make it a good counter but not a stomp for either lexorin users or receivers.